packages/realm-web-integration-tests/tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
 	"compilerOptions": {
 		"strict": true,
+		"sourceMap": true,
 		"module": "commonjs",
-		"target": "es6",
+		"target": "ES2019",
```

Comment:
Is this too modern? Do browsers and node.js support it?

Reply:
Thought about it, but I decided that I expect most users to pass the code exported by our package through their own bundler (Webpack / Parcel) invoking Babel to transpile it to their desired target. We should consider lowering the language version for the upcoming IIFE bundle.

Reply:
What do you think @steffenagger?

Reply:
What are the benefits of a higher target? Unless we want to expose features only found in "ES2019", I think we should go as low as makes sense, otherwise aren't we just limiting developers?
I skipped this one, as it was related to tests (I say "skipped", as I saw it, but haven't gotten to finish the review yet).

Reply:
This is in the integration tests, but I'm also increasing this for the main Realm Web package.
The benefit is a better developer experience (for me at least) and potentially better runtime performance as the browser implementation of the runtime would be used instead of TypeScript (or other) polyfills in situations where our users would actually be able to target a newer version of ES.

My thinking is - a user can always lower this version with Babel, if they run into problems, but they cannot increase the target if their environment allows it - if we've made the decision of targeting es6.

Reply:
I made this change to enable the use of `Object.fromEntries`, which is supported by 83.8% of users globally https://caniuse.com/#search=fromEntries and since I expect users of Realm Web to transpiling via a bundler, I don't think this will be an issue.

Reply:
Let's leave it as it is.